File: src/terceiro-bimestre/pfe/s18/a2/Usuario.js
class Usuario {
    constructor(id, nome, email, senha) {
        this.__id = id;
        this.__nome = nome;
        this.__email = email;
        this.__senha = senha;
    }

    getId() {
        return this.__id;
    }

    getNome() {
        return this.__nome;
    }

    getEmail() {
        return this.__email;
    }

    getSenha() {
        return this.__senha;
    }

    getInformacoes() {
        console.log(`Id: ${this.__id}`);
        console.log(`Nome: ${this.__nome}`);
        console.log(`Email: ${this.__email}`);
        //console.log(`Senha: ${this.__senha}`);
    }

}
export default Usuario;

File: src/quarto-bimestre/logica/biblioteca/Usuario.py
class Usuario:
    def __init__(self, id, nome, email, senha):
        self.id = id
        self.nome = nome
        self.email = email
        self.senha = senha

    def get_id(self):
        return self.id
    
    def get_nome(self):
        return self.nome
    
    def get_email(self):
        return self.email
    
    def get_senha(self):
        return self.senha

    def mostra_usuario(self):
        print(f"ID: {self.get_id()}")
        print(f"Nome: {self.get_nome()}")
        print(f"Email: {self.get_email()}")
        
                